Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameHotkeys.txt

```diff
@@ -517,4 +517,9 @@ Button/Hotkey/AP_zergmissileweapons5=A
 Button/Hotkey/AP_MercInfantry=B
 Button/Hotkey/AP_MercVehicle=F
 Button/Hotkey/AP_MercShip=S
+Button/Hotkey/AP_ZergMercBroodWar=W

```

Comment:
Pick an another hotkey - clashes with select Warpgate global hotkey

Reply:
Changed, didn't think this would matter since zerg vs protoss.  Is there any other global hot keys to avoid, or is there a specific list to check against?  I just looked up this, and I'm guessing it's the in-game hotkeys from this?: https://liquipedia.net/starcraft2/Hotkeys#In-Game_hotkeys